gcp/modules/backup-exporter/main.tf

```diff
@@ -4,6 +4,8 @@ terraform {
 
```

Comment:
Has this module already been applied anywhere? It seems to me it will suffer with same omission of the provider config block issue )https://github.com/gpii-ops/gpii-infra/blob/c2ef3a02cdccfd6a0e2c2bdfe9137bc749f01e6c/common/modules/backup-storage-bucket/main.tf#L10-L12).

Reply:
yes, it has been applied in common/stg

Reply:
Solved below